### PR TITLE
Update based on the new task description

### PR DESCRIPTION
**Pull request by NinaWan**
I haven't found the textpro.pl file, so I'm sure if I need to replace the argument - 'ita' in the last line with $langCode. The textpro.pl is from TextPro – FBK?

### CODE REVIEW

**Comment by marfox** [comment]
Yes, don't worry about that, it's part of an experiment for Italian.

File: extract_verbs.sh [hunk 1]
```diff
@@ -3,20 +3,91 @@
...
+
+# Check if the argument is null
+if [ ! -n "$1" ] ;then
+    echo "No language code entered!"
+    echo "Usage:"
+    echo "./extract_verbs.sh languageCode(e.g it OR italian for Italian)"   
+    exit
+fi
+# Get the entered language code
+langCode=$1
```
Comment (marfox) [inline extract_verbs.sh]: Just a minor comment: if you make this variable uppercase, it will improve the code readability

File: extract_verbs.sh [hunk 2]
```diff
@@ -3,20 +3,91 @@
...
+# Get the right tree-tagger file and abbreviative language code based on the input 
+case $langCode in
+	[iI][tT] | [iI][tT][aA][lL][iI][aA][nN] )
+		langCode="it"
+		treeTagger="tree-tagger-italian";;
```
Comment (marfox) [inline extract_verbs.sh]: and also this variable

**Comment by NinaWan** [comment]
Hi @marfox, I have capitalized the variables. Should I move to the next issue?

**Comment by marfox** [comment]
Sure, go ahead!
And thanks for the contribution!
BTW, you should introduce yourself in our GSoC mailing list. See http://wiki.dbpedia.org/gsoc2015/ideas

**Comment by NinaWan** [comment]
Got it. Thanks! @marfox

**Pull request merged**